tidb/pom.xml

```diff
@@ -21,6 +21,7 @@
     <protobuf.version>3.5.1</protobuf.version>
     <grpc.version>1.38.0</grpc.version>
     <proto.folder>${basedir}/proto</proto.folder>
+    <argLine>-Dfile.encoding=UTF-8 -Duser.timezone=GMT+8</argLine>
```

Comment:
does specifying a time zone in pom may cause the error for the user of another time zone?